module/VuFindApi/config/module.config.php

```diff
@@ -52,6 +55,28 @@
                         'action'     => 'record',
                     ]
                 ]
+            ],
+            'search2Apiv1' => [
+                'type' => 'Laminas\Router\Http\Literal',
+                'verb' => 'get,post,options',
+                'options' => [
+                    'route'    => '/api/v1/search2',
```

Comment:
How about using something like '/api/v1/index2/search' and '/api/v1/index2/record' as the routes? This would allow the endpoints to look to the same for any other indexes and operations too, and I feel that it would be better to avoid exposing the "search2" name.

Reply:
Does make sense, I changed it in follow-up